cloudant-client/src/main/java/com/cloudant/client/internal/views/CommonViewRequestBuilder.java

```diff
@@ -105,6 +105,12 @@ public RB reduce(boolean reduce) {
         return returnThis();
     }
 
+    @Override
+    public RB stable(String stable) {
+        viewQueryParameters.setStable(stable);
+        return returnThis();
+    }
+
     @Override
     public RB stale(String stale) {
```

Comment:
I think we could reasonably deprecate this method pointing to `stable` and `update` instead.

Reply:
Done in 1dceeae3f4050abe9c6909eb1cb4df9be959f62b